ecmd-core/pyapi/ecmdClientPyapi.i

```diff
@@ -12,13 +12,12 @@
 %include stdint.i
 %include ecmdCommon.i
 %include ecmdConst.i
-%apply uint32_t &OUTPUT { uint32_t & };
-//typemap for ecmdClockFreqMode_t input value passed by reference
-%typemap(in) ecmdClockFreqMode_t & (ecmdClockFreqMode_t temp) { temp = (ecmdClockFreqMode_t) PyInt_AsLong($input); $1 = &temp; }
 /*********** End Typemaps ***********/
 
 /*********** Start Applies ***********/
 // These are used to map C types that swig doesn't understand to types swig does understand
+%apply uint32_t &OUTPUT { uint32_t & };
+%apply int &OUTPUT { enum SWIGTYPE & };
```

Comment:
This is affecting this poorly defined functions like
ecmdGetClockSpeedSensor(..., ecmdClockSpeedType_t & i_clockspeedUnit, ...)